js-client-examples/browser-example/package.json

```diff
@@ -27,8 +26,8 @@
         "test": "jest --config=jest.config.js",
         "_test": "react-scripts test",
         "eject": "react-scripts eject",
-        "compile-aqua": "fluence aqua -i ./aqua/ -o ./src/_aqua",
-        "watch-aqua": "fluence aqua -w -i ./aqua/ -o ./src/_aqua"
+        "compile-aqua": "fluence aqua -i ./aqua/ -o src/aqua",
+        "watch-aqua": "fluence aqua -w -i ./aqua/ -o src/aqua"
```

Comment:
Maybe remove these? Since the project should be under `fcli`, it doesn't really need that anymore.